io.github.lainsce.Quilter.json

```diff
@@ -0,0 +1,106 @@
+{
+    "app-id" : "io.github.lainsce.Quilter",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "3.38",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "io.github.lainsce.Quilter",
+    "rename-icon" : "io.github.lainsce.Quilter",
+    "finish-args" : [
+        "--filesystem=xdg-data/io.github.lainsce.Quilter/",
+        "--filesystem=xdg-documents",
+        "--filesystem=xdg-download",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=network",
+        "--share=ipc"
+    ],
+    "cleanup" : [
+        "/cache",
+        "/man",
+        "/share/aclocal",
+        "/share/devhelp",
+        "/lib/systemd",
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/libvala*",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/vala",
+        "/bin/vapi*",
+        "*.a",
+        "*.la"
+    ],
+    "modules" : [
+        {
+            "name": "gtksourceview",
+            "buildsystem": "meson",
+            "sources": [{
+                "type": "archive",
+                "url": "https://download.gnome.org/sources/gtksourceview/4.7/gtksourceview-4.7.90.tar.xz",
+                "sha256": "983bdcb88f98285b3b997c9335057c8fbc45aad0a7c13cb65eb365567e26baaf"
+            }]
+        },
```

Comment:
gtksourceview4 is already on the sdk

Reply:
Oh, I see. Going to remove these lines then.